src/System.Private.CoreLib/shared/System/Globalization/CultureInfo.cs

```diff
@@ -1287,24 +1175,53 @@ public static CultureInfo GetCultureInfo(string name)
         /// </summary>
         public static CultureInfo GetCultureInfo(string name, string altName)
         {
-            if (name == null)
+            if (name is null)
             {
                 throw new ArgumentNullException(nameof(name));
             }
-            if (altName == null)
+            if (altName is null)
```

Comment:
just curious, how are `== null` and `is null` different?

Reply:
They're the same if the type in question doesn't overload operator==.  If the type does overload operator==, they're different in that `is null` doesn't pay attention to it.